dbcs/honda_crv_touring_2016_can.dbc

```diff
@@ -0,0 +1,312 @@
+VERSION ""
```

Comment:
There's surprisingly little public information about it, but there's a library in openpilot to read it and it is the industry standard.